src/Database/Reader.php

```diff
@@ -110,6 +110,26 @@ public function anonymousIp($ipAddress)
     }
 
     /**
+     * This method returns a GeoIP2 ASN model.
+     *
+     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     *
+     * @return \GeoIp2\Model\Asn
+     *
+     * @throws \GeoIp2\Exception\AddressNotFoundException if the address is
```

Comment:
There is [this test](https://github.com/maxmind/GeoIP2-php/blob/24b278b0195d7067e930b844c44faf8fae5bdbc7/tests/GeoIp2/Test/Database/ReaderTest.php#L50). All the database methods use the same code for throwing the exception, but it would be nice to parameterize the test so it tested all of them.